libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -280,7 +281,7 @@ namespace
 			json_obj["hostname"] = host_name;
 
 			pid_t pid = getpid();
-			json_obj["pid"] = std::to_string(pid);
+			json_obj["pid"] = pid;
```

Comment:
Could we do like line 372 here?
```suggestion
			json_obj["pid"] = getpid();
```
Looks like this may be done in a later commit, but I'll leave this here just in case.

Reply:
Yeah this is done in a later commit